cmd/allocator/main_test.go

```diff
@@ -48,6 +48,7 @@ func TestAllocateHandler(t *testing.T) {
 				},
 			}, nil
 		},
+		mTLSEnabled: true,
```

Comment:
If you change the default for the feature gate, you shouldn't need to add anything here to keep the old behavior. You will only need to make changes where you want to test the new behavior.

Reply:
Changing it to mTLSDisabled let me remove this.